modules/control/dag/mpc_module.dag

```diff
@@ -31,7 +28,6 @@ module_config {
         class_name : "MPCControllerSubmodule"
         config {
             name: "MPCControllerSubmodule"
-            config_file_path: "/apollo/modules/control/conf/calibration_table.pb.txt"
```

Comment:
@sjiang2018 Why remove the conf files here?

Reply:
All configs are included in mpc_controller conf file